server_installation/topics/cache/replication.adoc

```diff
@@ -23,3 +24,7 @@ Here we've changed it so at least two nodes will replicate one specific user log
 TIP:  The number of owners recommended is really dependent on your deployment.  If you do not care if users are logged
       out when a node goes down, then one owner is good enough and you will avoid replication.
 
+TIP:  It is very good if you configure your environment to use loadbalancer with sticky sessions. It is beneficial for performance
+      as {project_name} server, where the particular request is served, will be usually the owner of the data from distributed cache and
+      hence it will be able to lookup the data locally. More details in <<sticky-sessions>>.
+
```

Comment:
TIP:  It is generally wise to configure your environment to use loadbalancer with sticky sessions. It is beneficial for performance as {project_name} server, where the particular request is served, will be usually the owner of the data from the distributed cache and will therefore be able to look up the data locally. See <<sticky-sessions>> for more details.